test/object-expressions/object-expression.js

```diff
@@ -218,5 +218,79 @@ suite("Parser", function () {
         new Shift.ShorthandProperty(new Shift.Identifier("b")),
       ])
     );
+
+    testParse("({a(){}})", expr,
+      new Shift.ObjectExpression([
+        new Shift.Method(
+          false,
+          new Shift.StaticPropertyName("a"),
+          [],
+          null,
+          new Shift.FunctionBody([], []), false
+        )
+      ])
+    );
+    testParse("({a(){let a;}})", expr,
+      new Shift.ObjectExpression([
+        new Shift.Method(
+          false,
+          new Shift.StaticPropertyName("a"),
+          [],
+          null,
+          new Shift.FunctionBody(
+            [],
+            [new Shift.VariableDeclarationStatement(
```

Comment:
This array formatting is inconsistent with the rest of the codebase. Please make it consistent.